app/models/concerns/statuses.rb

```diff
@@ -0,0 +1,9 @@
+# Encoding: utf-8
+module Statuses
+  STATUSES = { 'opened' => 'Запит прийнято',
+               'pending' => 'Очікує на модерацію',
+               'declined' => 'Запит відхилено',
+               'closed' => 'Запит вирішено' }.freeze
+
+  STATUSES_SYM = [:pending, :declined, :opened, :closed].freeze
```

Comment:
Try
```ruby
module Statuses
STATUSES = { :opened => 'Запит прийнято',
               :pending => 'Очікує на модерацію',
               :declined => 'Запит відхилено',
               :closed => 'Запит вирішено' }.with_indifferent_access.freeze

  STATUSES_SYM = STATUSES.keys
```

Not sure if you need ```STATUSES_SYM``` though.
Also read about http://api.rubyonrails.org/v4.2/classes/ActiveSupport/HashWithIndifferentAccess.html